packages/flutter/lib/src/material/drop_down.dart

```diff
@@ -533,10 +538,15 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> {
           mainAxisAlignment: MainAxisAlignment.spaceBetween,
           mainAxisSize: MainAxisSize.min,
           children: <Widget>[
-            new IndexedStack(
-              index: _selectedIndex,
-              alignment: FractionalOffset.centerLeft,
-              children: config.items
+            // The button's size is defined by its largest menu item. If value is
+            // null then an item does not appear.
+            new Opacity(
```

Comment:
Why use an opacity here?  I guess that works, but I would have expected there to be a more direct way of making a subtree take up space but be invisible.

In particular, the children can still be hit with opacity 0.0, but presumably that's not what you want.